### PR TITLE
Convert hideFieldsOnNewIncident script to PY3

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Converted the hideFieldsOnNewIncident script to Py3:
- Split the script. 
- Added a unit test.
- Converted the script to Python3. 
- Deleted the old script. 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bziser — APPROVED** [review]

File: Packs/CommonScripts/Scripts/HideFieldsOnNewIncident/HideFieldsOnNewIncident.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        in the close form.
+
+        Args:
+            incident: Incident object
+            field: Field definition object
+
+        Returns: demisto.results object
+
+    """
+    if incident["id"] == "":
```
Comment (tkatzir) [inline Packs/CommonScripts/Scripts/HideFieldsOnNewIncident/HideFieldsOnNewIncident.py:20]:
```suggestion
    if 'id' not in incident:
```

**Review by tkatzir — COMMENTED** [review]
Looks good. Minor comment.

File: Packs/CommonScripts/Scripts/HideFieldsOnNewIncident/HideFieldsOnNewIncident.py (hunk 1 above)
Comment (ShacharKidor) [reply Packs/CommonScripts/Scripts/HideFieldsOnNewIncident/HideFieldsOnNewIncident.py]: Fixed - thank you :)

**Review by tkatzir — APPROVED** [review]

**Comment by ShacharKidor** [comment]
@ShahafBenYakir - Please force merge the PR - the validate error is because of a deletion of the old (unified) script.

**Pull request merged**